tests/test_check_prerequisites.py

```
@@ -293,3 +325,75 @@ def test_check_prerequisites_with_failing_callable():
+    # Mock that tracks when env is checked
+    class EnvTracker:
+        def __init__(self):
+            self.original_in = os.environ.__contains__
+
+        def __call__(self, key):
+            if key == "TEST_ENV" and "env_check" not in executed:
+                executed.append("env_check")
+            return self.original_in(key)
+
+    # Mock that tracks command execution
+    def mock_run(*args, **kwargs):
```

Ruff (0.13.3)

Unused function argument: `args`

(ARG001)